onchain/rollups/.changeset/sweet-pianos-exist.md

```diff
@@ -0,0 +1,10 @@
+---
+"@cartesi/rollups": major
+---
+
+The CartesiDApp contract was renamed as Application.
+The ICartesiDApp contract was renamed as IApplication.
+The CartesiDAppFactory contract was renamed as ApplicationFactory.
+The DAppAddressRelay contract was renamed as ApplicationAddressRelay.
+The encodeDAppAddressRelay function from the former DAppAddressRelay contract was renamed as encodeApplicationAddressRelay.
+The OnlyDApp error from the former CartesiDApp contract was renamed as OnlyApplication.
```

Comment:
Can we format these names as code, please?

Reply:
Right. For sure :)